src/api/auth/signup/signupUser.ts

```
@@ -0,0 +1,47 @@
+        nickName,
+        marketingReceive,
+      })
+    ).data;
+
+    if (findExCode(response.code)) {
+      throw new Error(response.message);
+    }
+
+    return response.data;
+  } catch (e) {
+    throw e;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)